### PR TITLE
Documentation for PTVS projects to use device name instead of IP addr…

### PR DESCRIPTION
**Pull request by juanyaw**
…ess due to an issue in remote debugging in PTVS release

### CODE REVIEW

File: win10/samples/Python.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ print("Hello, World from IoT!")
 
 * Right-click the Project node and select Properties
 
-* Make sure you set the 'Remote Machine' setting to point to your device. Go back to the basic 'Hello World' [sample]({{site.baseurl}}/win10/samples/HelloWorld.htm) if you need guidance.
+* Make sure you set the 'Remote Machine' setting to point to your device by machine name. Go back to the basic 'Hello World' [sample]({{site.baseurl}}/win10/samples/HelloWorld.htm) if you need guidance.
+ <br />**NOTE: For now, use device name in lieu of IP address.  If the device name is not unique, use <code>setcomputername</code> tool on the device to reset the device name and reboot.  Documentation for this can be found [here]({{site.baseurl}}/win10/samples/Powershell.htm)**
```
Comment (jongio) [inline win10/samples/Python.md]: Needs to be PowerShell.htm, not Powershell.htm

File: win10/samples/Python.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ print("Hello, World from IoT!")
...
 
 * When everything is set up, you should be able to press F5 from Visual Studio: The Python app will deploy and start on the device.
-
-* **`NOTE: Launching the debugger for Python can take a few minutes to connect and start debugging the remote Python.  If it takes more than a few minutes, there may be a problem with msvsmon on the remote device.  Please check the FAQ for any potential fixes/workarounds.`**
+ <br />**NOTE: Launching the debugger for Python can take a few minutes to connect and start debugging the remote Python.  If it takes more than a few minutes, there may be a problem with msvsmon on the remote device.  Please check the FAQ for any potential fixes/workarounds.**
```
Comment (jongio) [inline win10/samples/Python.md]: Please don't use HTML

File: win10/samples/PythonBlinky.md [hunk 3]
```diff
@@ -72,10 +72,10 @@ Remember that we connected the other end of the LED to the 3.3 Volts power suppl
 * Right-click the Project node and select Properties
 
 * Make sure you set the 'Remote Machine' setting to point to your device. Go back to the basic Python [sample](Python.htm) if you need guidance.
+ <br />**NOTE: For now, use device name in lieu of IP address.  If the device name is not unique, use <code>setcomputername</code> tool on the device to reset the device name and reboot.  Documentation for this can be found [here]({{site.baseurl}}/win10/samples/Powershell.htm)**
```
Comment (jongio) [inline win10/samples/PythonBlinky.md]: Not HTML

Comment (jongio) [inline win10/samples/PythonBlinky.md]: Case of PowerShell.htm

File: win10/samples/PythonBlinky.md [hunk 4]
```diff
@@ -72,10 +72,10 @@ Remember that we connected the other end of the LED to the 3.3 Volts power suppl
...
 
 * When everything is set up, you should be able to press F5 from Visual Studio: The Python app will deploy and start on the device.
-
-* **`NOTE: Launching the debugger for Python can take a few minutes to connect and start debugging the remote Python.  If it takes more than a few minutes, there may be a problem with msvsmon on the remote device.  Please check the FAQ for any potential fixes/workarounds.`**
+ <br />**NOTE: Launching the debugger for Python can take a few minutes to connect and start debugging the remote Python.  If it takes more than a few minutes, there may be a problem with msvsmon on the remote device.  Please check the FAQ for any potential fixes/workarounds.**
```
Comment (jongio) [inline win10/samples/PythonBlinky.md]: No HTML

File: win10/samples/PythonBlinkyServer.md [hunk 5]
```diff
@@ -82,10 +82,10 @@ Remember that we connected the other end of the LED to the 3.3 Volts power suppl
 * Right-click the Project node and select Properties
 
 * Make sure you set the 'Remote Machine' setting to point to your device. Go back to the basic Python [sample]({{site.baseurl}}/win10/samples/Python.htm) if you need guidance.
+ <br />**NOTE: For now, use device name in lieu of IP address.  If the device name is not unique, use <code>setcomputername</code> tool on the device to reset the device name and reboot.  Documentation for this can be found [here]({{site.baseurl}}/win10/samples/Powershell.htm)**
```
Comment (jongio) [inline win10/samples/PythonBlinkyServer.md]: HTML, Case

File: win10/samples/PythonBlinkyServer.md [hunk 6]
```diff
@@ -82,10 +82,10 @@ Remember that we connected the other end of the LED to the 3.3 Volts power suppl
...
 
 * When everything is set up, you should be able to press F5 from Visual Studio: The Python app will deploy and start on the device.
-
-* **`NOTE: Launching the debugger for Python can take a few minutes to connect and start debugging the remote Python.  If it takes more than a few minutes, there may be a problem with msvsmon on the remote device.   Please check the FAQ for any potential fixes/workarounds.`**
+ <br />**NOTE: Launching the debugger for Python can take a few minutes to connect and start debugging the remote Python.  If it takes more than a few minutes, there may be a problem with msvsmon on the remote device.  Please check the FAQ for any potential fixes/workarounds.**
```
Comment (jongio) [inline win10/samples/PythonBlinkyServer.md]: HTML

**Comment by juanyaw** [comment]
Fixed by removing HTML comments and updating PowerShell link

**Pull request merged**